OSIsoft_Demo/src/OSIsoftServer.java

```diff
@@ -53,4 +61,29 @@ private static String buildBody(String value) {
       return jsonBody;
    }
    
+   // Reads a file and returns it as a string
+   public static String readFile(String path) {

```

Comment:
Should probably document that upon any error returns an empty string. Its tough to know how, as the caller, there was an empty file though vs. an error.

Reply:
Documented in the method